examples/tf-azurerm-vnet/outputs.tf

```diff
@@ -18,7 +18,10 @@ output "vnet_dns_servers" {
   value = "${azurerm_virtual_network.vnet.dns_servers}"
 }
 
-output "vnet_subnet" {
+output "vnet_subnets" {
   value = "${azurerm_virtual_network.vnet.subnet}"
```

Comment:
I don't think this will work in it's current form, instead I think this wants to return a list of subnet ID's via:

```
value = "${azurerm_subnet.subnet.*.id}"
```

What do you think?